CMakeLists.txt

```diff
@@ -20,17 +20,14 @@
 
 # Main header: Project name & Configuration
 # ------------------------------------------------------------------------------------------------------
-CMAKE_MINIMUM_REQUIRED(VERSION 2.8.11)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.12)
 
 SET(PROJ_MAIN_NAME "tutorial")
 PROJECT(${PROJ_MAIN_NAME})
 
 
 # Configuration types
 SET(CMAKE_CONFIGURATION_TYPES "Debug;Release" CACHE STRING "Configs" FORCE)
-IF(DEFINED CMAKE_BUILD_TYPE AND CMAKE_VERSION VERSION_GREATER "2.8")
```

Comment:
Only delete the version check of the if statement. The rest of it should stay.

Reply:
Done